src/js/claims-status/containers/DocumentRequestPage.jsx

```diff
@@ -46,6 +46,9 @@ class DocumentRequestPage extends React.Component {
     }
   }
   componentWillReceiveProps(props) {
+    if (props.loading && !!props.trackedItem) {
```

Comment:
I think this should be `if (!props.loading && !!props.trackedItem) {`

Reply:
Thanks @jbalboni, you're right, the boolean clauses were reversed, changes made.